enhancements/install/user-provided-vpcs.md

```diff
@@ -0,0 +1,184 @@
+---
+title: user-provided-vpcs-on-aws
```

Comment:
I don't think you meant to include this here?

Reply:
nope.  Removed with eabf021 -> 3fe8a38.